### PR TITLE
[NEW] Commands

### PR DESCRIPTION
**Pull request by mstuttgart**
* Added snippets to *create* and *write* and *onchange* methods .
* Added snippet to *group* tag in xml.
* Added snippet to *button box (smart button)* and buttons in xml.
* Added snippet to *fields* and *field 'state' (with it attributes) in xml;
* Import Odoo snippet in alphabetical order

### CODE REVIEW

**Comment by mstuttgart** [comment]
@mohamedmagdy

File: Odoo.xml [hunk 1]
```diff
@@ -57,62 +57,62 @@
       <option name="Python" value="true" />
     </context>
   </template>
-  <template name="odoo_field_integer" value="new_field = fields.Integer(string=&quot;&quot;, required=False, )" description="Add new field of type Integer" toReformat="false" toShortenFQNames="true">
+  <template name="odoo_field_integer" value="new_field = fields.Integer(string='', required=False, )" description="Add new field of type Integer" toReformat="false" toShortenFQNames="true">
```
Comment (mohamedmagdy) [inline Odoo.xml]: Great work, @mstuttgart . 
Can you help me to understand why you replaced the double qoutes to be the single quote !!

Thank you :)

Comment (mstuttgart) [reply Odoo.xml]: It is only as visual improvement. A guess the python code more clean when use single quotes. 

I can turn back it if you dont like.

**Review by mohamedmagdy — APPROVED** [review]

**Review by mohamedmagdy — CHANGES_REQUESTED** [review]
@mstuttgart , You are doing a great job and I really appreciate it.
Well, in most cases Odoo is using the double quotes and in few times it uses single quotes. So, I think it will be easier for the odoo developers to see it in double quotes, What do you think?

**Comment by mstuttgart** [comment]
@mohamedmagdy no problem! I update the commits history and remove this commit. 

I added snippets to button, button-box, field and field 'state' in XML.

**Review by mohamedmagdy — APPROVED** [review]
Thank you, @mstuttgart for being active :)
I am accepting this pull request with pleasure :+1:

**Comment by mohamedmagdy** [comment]
@mstuttgart , as you can see there is a conflict. It will be nice of you to solve it whenever you have time.

Thank you :)

**Comment by mstuttgart** [comment]
@mohamedmagdy fixed! Thanks! :100:

**Comment by mohamedmagdy** [comment]
@mstuttgart , no it is me who should thank you :)

**Pull request merged**